zulipterminal/ui_tools/views.py

```diff
@@ -880,8 +880,11 @@ def keypress(self, size: urwid_Size, key: str) -> str:
 class StreamInfoView(PopUpView):
     def __init__(self, controller: Any, color: str,
                  name: str, desc: str) -> None:
-        # TODO: Width & Height handling could be improved
-        self.width = max(len(desc), len("# {}".format(name))) + 2
+        # TODO: Height handling could be improved
+        # Add 4 (for 2 Unicode characters on either side) to the popup title
+        # length to make sure that the title gets displayed even when the
+        # content is shorter than the title length (+4 Unicode characters).
+        self.width = max(len(desc), len("# {}".format(name)) + 4)
```

Comment:
Yes, it is how the bug gets fixed.

Reply:
Yes, that is how the bug gets fixed.

Reply:
Well, this has two effects, as I understand it:
* it fixes the bug with an empty description, where the title isn't listed (?)
* it removes a border of one space either side where there are descriptions

I understand that this is a bugfix for the former, but the latter isn't obvious.

To clarify, we don't always have left/right padding in popups, but I'm not sure if we want to change this behavior for now?

Reply:
I understand the concern now. I will edit the commit to only do the former i.e. the bugfix. However, since the help menu doesn't add extra spaces, the two extra spaces will eventually get removed in the refactoring commit (`chart_widths`). Would that be okay?